fvtest/compilertest/linter.mk

```diff
@@ -150,11 +150,11 @@ linter:: omrchecker
 # however, that doesn't fix all of our problems, as we are typically not
 # reconfiguring for clang when running lint (though, perhaps this is an early
 # indicator that we really should be.  
-BLACKLISTED_DEFINES='-D__sync()=' '-D__lwsync()=' '-D__isync()='
+BLOCKLISTED_DEFINES='-D__sync()=' '-D__lwsync()=' '-D__isync()='
```

Comment:
I don't think "block" is the right term here because the defines aren't really "blocked." Maybe `EXCLUDED_DEFINES` is more appropriate?